cloud/pkg/cloudhub/handler/messagehandler.go

```diff
@@ -529,22 +538,36 @@ func (mh *MessageHandle) saveSuccessPoint(msg *beehiveModel.Message, info *model
 				klog.Errorf("failed to delete message %v, err: %v", msg, err)
 				return
 			}
-			mh.deleteSuccessPoint(resourceNamespace, objectSyncName)
+			_, err := mh.crdClient.ReliablesyncsV1alpha1().ObjectSyncs(resourceNamespace).Get(context.Background(), objectSyncName, metav1.GetOptions{})
+			if err != nil && apierrors.IsNotFound(err) {
+				klog.Infof("ObjectSync: %v has already been deleted", objectSyncName)
+				mh.deleteSuccessPoint(resourceNamespace, objectSyncName)
```

Comment:
This logic seems to be reversed.
```go
_, err := mh.crdClient.ReliablesyncsV1alpha1().ObjectSyncs(resourceNamespace).Get(context.Background(), objectSyncName, metav1.GetOptions{})<br class="Apple-interchange-newline">
if err != nil && apierrors.IsNotFound(err) {
	klog.Infof("ObjectSync: %v has already been deleted", objectSyncName)
        return
}
mh.deleteSuccessPoint(resourceNamespace, objectSyncName)
return
```
But I suggest just adding an error judgment to the `mh.deleteSuccessPoint` instead of getting it once first.
```go
func (mh *MessageHandle) deleteSuccessPoint(resourceNamespace, objectSyncName string) {
        err := mh.crdClient.ReliablesyncsV1alpha1().ObjectSyncs(resourceNamespace).Delete(context.Background(), objectSyncName, *metav1.NewDeleteOptions(0))
        if err != nil && !apierrors.IsNotFound(err) {
                klog.Errorf("Delete Success Point failed with error: %v", err)
        }
}
```